releasenotes/notes/asm-add-http-route-for-flask-8b13c721e2f7543b.yaml

```diff
@@ -1,4 +1,4 @@
 ---
 other:
   - |
-    ASM: add http.route for Python Flask.
\ No newline at end of file
+    ASM: add http.route tag to root span for Flask framework.
```

Comment:
is this for ASM or for tracing?

Reply:
This change technically applies to the tracer in general. However, it was an ASM-related change so unclear which avenue we should take.

Reply:
I think since this was a change to the span tags for the Flask tracing integration it should be listed as a change to "tracing".

The fact that the change was done by ASM or for ASM shouldn't matter here.

Unless the change is actually "ASM now uses the http.route tag to perform some cool automated security checks"